x-pack/plugins/elastic_assistant/server/routes/request_context_factory.ts

```diff
@@ -32,15 +32,18 @@ interface ConstructorOptions {
   plugins: ElasticAssistantPluginSetupDependencies;
   kibanaVersion: string;
   assistantService: AIAssistantService;
+  // attackDiscoveryTask: AttackDiscoveryTask;
```

Comment:
nit: consider removing the commented code on lines `35`, `41`, `46`, `69`, `70`

Reply:
ah thanks!